test/test_ops.py

```diff
@@ -193,6 +193,7 @@ def test_multiple_devices(self, devices, dtype, op):
 
     # Tests that the function and its (ndarray-accepting) reference produce the same
     #   values on the tensors from sample_inputs func for the corresponding op.
+    @unittest.skipIf(TEST_WITH_ROCM, "Skipping as we won't be fixing test_reference_testing_linalg_tensorinv_cuda_float32 test failure on an older version of pytorch")
```

Comment:
@rraminen You missed an `import unittest` in this file. Without that, we get a `name 'unittest' is not defined` error. Fixed in https://github.com/ROCm/pytorch/commit/dcbc5703cea53af17c9e7cdcf6a00b5be0589044

Reply:
Although, on further investigation, this is an incorrect way of skipping these unit tests. Proper fix in https://github.com/ROCm/pytorch/commit/514d6738d436124ced368a3b809c2b09692a0c05